subsys/zigbee/lib/zigbee_fota/src/zigbee_fota.c

```diff
@@ -72,6 +73,7 @@ union zb_ota_app_ver {
 	};
 };
 
+static uint8_t zb_ota_buf[CONFIG_ZIGBEE_FOTA_DATA_BLOCK_SIZE];
```

Comment:
Does it make sense to use this kconfig for the size of the `ota_buf` ? I see its used when declaring some `OTA` related attributes, so just double checking :)

Reply:
The `CONFIG_ZIGBEE_FOTA_DATA_BLOCK_SIZE` defines the maximum size of the new firmware chunk that a Zigbee device can accept through the Zigbee network.
Theoretically - a user may want to reduce this value to reduce the RX time, but that makes the OTA process longer, leading to more wake-ups. Unfortunately, I am not aware of real-life measurements, that would allow us to conclude, what is the optimal value to get the lowest possible energy consumption.